src/validators/uv_check.py

```diff
@@ -0,0 +1,67 @@
+"""UV Check Validator."""
+
+import logging
+from typing import List, Tuple
+
+import bpy
+
+from . import BaseValidator, Category
+
+
+def _get_multiple_uv_models(models: List[str]) -> List[str]:
+    return [
+        model for model in models
+        if (
+            bpy.data.objects[model].data is not None
+            and hasattr(bpy.data.objects[model].data, 'uv_layers')  # noqa: WPS421
```

Comment:
Por que usar `hasattr` ao invés de `in` nesse caso? Dúvida real sobre python haha.

Reply:
O `in` só funciona para dicionários/listas/sets (tudo que herda de `Collection`, acho). Como o data é um objeto, objetos sem esse atributo, como o `Armature` provocariam erro aqui